code/obj/critter/pets_small_animals.dm

```diff
@@ -511,8 +510,7 @@
 			src.visible_message("<span class='combat'><b>[user]</b> pets [src]!</span>")
 			if(prob(30))
 				src.icon_state = "[src.doggy]-lying"
-				for(var/mob/O in hearers(src, null))
-					O.show_message("<span class='notice'><B>[src]</B> flops on his back! Scratch that belly!</span>",2)
+				src.audible_message("<span class='notice'><B>[src]</B> flops on his back! Scratch that belly!</span>",2)
```

Comment:
wait why was this checking hearers in the first place. This should be a visible message 😱